content/desktop/release-notes.md

```diff
@@ -24,11 +24,71 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/releases.md).
 
-## 4.24.2
+## 4.25.0
+
+{{< release-date date="2023-10-26" >}}
+
+{{< desktop-install all=true version="4.25.0" build_path="/" >}}
+
+### Upgrades
+
+- [Compose v2.23.0](https://github.com/docker/compose/releases/tag/v2.23.0)
+- [Docker Scout CLI v1.0.9](https://github.com/docker/scout-cli/releases/tag/v1.0.9).
+- [Kubernetes v1.28.2](https://github.com/kubernetes/kubernetes/releases/tag/v1.28.2)
+- [cri-dockerd v0.3.4](https://github.com/Mirantis/cri-dockerd/releases/tag/v0.3.4)
+- [CNI plugins v1.3.0](https://github.com/containernetworking/plugins/releases/tag/v1.3.0)
+- [cri-tools v1.28.0](https://github.com/kubernetes-sigs/cri-tools/releases/tag/v1.28.0)
+### New
+
+- Rosetta is now available for all users on MacOS 13 or greater, providing faster emulation of Intel-based images on Apple Silicon.
+- Docker Desktop will now detect if a WSL version is out of date. If an out dated version of WSL is detected, you can now optionally allow Docker Desktop automatically updates the installation or manually update WSL outside of Docker Desktop.
+- Installation pre-requisites for Windows is now higher than 19043 for both GAC and LTSC.
+- Prevent Docker Desktop users from enabling Docker Scout image analysis via settings in `admin-settings.json`.
```

Comment:
```suggestion
- Administrators can now prevent their users from enabling Docker Scout image analysis in [Settings Management](hardened-desktop/settings-management/configure.md) 
```

Reply:
How about using "ability to control" instead of "prevent users from enabling" (because admin-setting can be used to prevent users from disabling the feature as well).
cc @KatTomrushka

Reply:
`Administrators now have the ability to control Docker Scout image analysis in Settings Management`